src/app/components/listing/details/compliance/direct-reviews.jsx

```diff
@@ -64,6 +65,10 @@ const useStyles = makeStyles({
       flexDirection: 'row',
     },
   },
+  errorChip: {
+    color: `${palette.white}`,
+    backgroundColor: `${palette.error}`,
```

Comment:
These should be un-interpolated as well. It's necessary to do this if there are multiple values in this row, like on line 52, where the `backgroundColor` has both a color and the !important tag, but if it's not needed, we shouldn't do it

Reply:
Thanks for the clarification!